### PR TITLE
e2e: move funs of framework/deployment/fixtures.go to e2e/apps/deploy…

### PR DESCRIPTION
**Pull request by yuxiaobo96**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

When I read the issue (#86052 ) example (#86050), I found that there was a stand-up function that had not been migrated.

**Which issue(s) this PR fixes**:

Ref: #86052 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/apps/deployment.go [hunk 1]
```diff
@@ -1122,3 +1122,8 @@ func waitForReplicaSetTargetSpecReplicas(c clientset.Interface, replicaSet *apps
 	}
 	return err
 }
+
+// CheckDeploymentRevisionAndImage checks if the input deployment's and its new replica set's revision and image are as expected.
+func CheckDeploymentRevisionAndImage(c clientset.Interface, ns, deploymentName, revision, image string) error {
```
Comment (oomichi) [inline test/e2e/apps/deployment.go]: Need to make the function name lower as local function like `checkDeploymentRevisionAndImage` .

Comment (yuxiaobo96) [reply test/e2e/apps/deployment.go]: done

Comment (yuxiaobo96) [reply test/e2e/apps/deployment.go]: @oomichi 
But the test failed again. I don't understand why the test failed, please help me.
Thanks!

**Comment by yuxiaobo96** [comment]
@oomichi 
The test has been passed.
I will squash these commits later, thanks for your advice.

**Review by SataQiu — APPROVED** [review]
Thanks @yuxiaobo96 
/approve
/lgtm

**Pull request merged**